lms/envs/common.py

```diff
@@ -2535,6 +2538,20 @@ def _make_locale_paths(settings):
     CORS_ORIGIN_WHITELIST = ()
     CORS_ORIGIN_ALLOW_ALL = False
 
+# Edraak: x-api-accept-language header for APIs
```

Comment:
Omit the defaults by using the following:

```
from corsheaders.defaults import default_headers
CORS_ALLOW_HEADERS =  default_headers + (
    'accept-language',
    'x-api-accept-language'
)
```

Reply:
Done, but I used list syntax